### PR TITLE
Improve discoverability of napari/docs repository.

### PR DESCRIPTION
**Pull request by Czaki**
# Fixes/Closes

# Description

Currently, When user see list of issue templates, there is a Documentation template that suggest that all documentation-related problems should be reported in the napari main repository. 

In This PR I add information about `napari/docs` 

# References

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: "\U0001F4DA Documentation"
-about: Report an issue with napari documentation
+about: Report an issue with napari code documentation
```
Comment (melissawm) [inline .github/ISSUE_TEMPLATE/documentation.md:3]: I'd say not only *code* documentation, but all documentation should be reported there.

Comment (Czaki) [reply .github/ISSUE_TEMPLATE/documentation.md:3]: But most of the documentation does not live there. Source files are in the docs repository.

Comment (melissawm) [reply .github/ISSUE_TEMPLATE/documentation.md:3]: Oh sorry, I read that wrong - nevermind! 😄

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
 blank_issues_enabled: true  # default
 contact_links:
+- name: 🗎 napari documentation
+  url: https://github.com/napari/docs/issues
+  about: |
+    If you found some problem with content on the napari documentation site, or miss rendering of some content, please let us know here.
```
Comment (psobolewskiPhD) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    If you've found a problem with content on the napari documentation site (napari.org) or with the rendering of the content, please let us know here.
```

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by psobolewskiPhD** [comment]
Good catch! Thanks @Czaki ! ❤️ 
I made a small tweak to the wording, not sure it's clearer or not.

**Pull request merged**